main.go

```diff
@@ -35,6 +37,16 @@ func main() {
 	stage := rootCmd.PersistentFlags().String("stage", "", "Stage for your service.")
 	profile := rootCmd.PersistentFlags().String("profile", "", "Janus profile for your service (when running locally).")
 
+	secretRecoveryWindow := rootCmd.PersistentFlags().Int64("secret-recovery-window", 30, "Recovery window when deleting secrets from Secret Manager, in days. Set to 0 to disable recovery (use with caution!)")
+	timeout := rootCmd.PersistentFlags().Int64("timeout", 5, "Number of seconds to wait for AWS operations")
+
+	argConf := config.Config{App: *app, Stack: *stack, Stage: *stage}
+	conf, configErr := config.Read(argConf, config.DefaultFiles()...)
```

Comment:
Slightly torn on this. It avoids the repetition but also separates the error handling from the error definition, which is potentially confusing and error-prone in the future - as it makes it easier not to handle the error correctly.